onnxruntime/test/providers/nnapi/nnapi_basic_test.cc

```diff
@@ -492,6 +492,22 @@ TEST(NnapiExecutionProviderTest, TestOrtFormatModel) {
 #endif
 }
 
+// test that NNAPI EP can process an activation node that is outside of its partition
+TEST(NnapiExecutionProviderTest, ActivationOutsideOfPartition) {
```

Comment:
this fails the NNAPI EP reduced ops CI build. the [ops and types config file](https://github.com/microsoft/onnxruntime/blob/efb0928e2bba9d6e942abaad63a888a75dfc976f/onnxruntime/test/testdata/required_ops_and_types.config) doesn't contain the Transpose op which is inserted. a simple fix is to add Transpose to the config file.
however, we probably need a better way to handle ops inserted by layout transformation (e.g., Transpose) as well as saved runtime optimizations. ops for the former could be always enabled and the config file creation can be updated to account for the latter.

Reply:
we'd need a flag to enable including them so that in scenarios where you're not using runtime optimizations/layout transformations and you need the smallest binary size possible we're not including unnecessary things.

they don't necessarily have to be in the config though as we do the reduction as part of the build (when run from build.py), and we'd know if nnapi or runtime optimization was enabled at that point.

Reply:
Short term can we update the config file used in the test (can add a separate section with comments as there can be multiple entries for an opset), and address the need to be able to include additional kernels in the build separately?